events-service/engine.py

```diff
@@ -17,9 +18,106 @@
 if RESET_DB:
     Base.metadata.drop_all(engine)
     Base.metadata.create_all(engine)
+    # Create a publisher factory.
+    publisher_factory = pubsub.PublisherFactory()
+
+    # Create a publisher.
+    publisher = publisher_factory.create_publisher(topic_name='events-mockdata')
 
     with Session(engine) as session:
 
         #TODO: Write a script to populate the database with some users
+        event1 = Events("Esports Tournament: League of Legends", "https://example.com/league_of_legends_thumbnail.jpg", "Compete in our annual League of Legends tournament and prove your skills!", "https://example.com/league_of_legends_tournament", ["gaming", "esports", "League of Legends"], 'Wed, 27 July 2016 07:45:00 UTC', "upcoming", "Gamer's Arena", ["Cash Prize", "Exclusive In-Game Items"])
```

Comment:
can you change it so that you use kwargs like such:
```
event1 = Events(title="Esports Tournament: League of Legends",
                url="https://example.com/league_of_legends_thumbnail.jpg",
                ...)
```